examples/Example1.cpp

```diff
@@ -24,11 +24,11 @@
 //
 //
 //  This piece of code was developed as part of ICE-DIP project at CERN.
-//  "ICE-DIP is a European Industrial Doctorate project funded by the European Community's 
+//  "ICE-DIP is a European Industrial Doctorate project funded by the European Community's
 //  7th Framework programme Marie Curie Actions under grant PITN-GA-2012-316596".
 //
 
-#include "../UMESimd.h"
```

Comment:
This violates the convention of all files named using 'UME' prefix.
I also think it is unnecessary to add addditional include file in this case, as it doesn't add anything useful. Move UMESimd.h back to top directory and leave it there.